travis.sh

```diff
@@ -108,7 +108,7 @@ sudo -E sh -c 'echo "deb $ROS_REPOSITORY_PATH `lsb_release -cs` main" > /etc/apt
 # apt key acquisition. Since keyserver may often become accessible, backup method is added.
 sudo apt-key adv --keyserver $APTKEY_STORE_SKS --recv-key $HASHKEY_SKS || ((echo 'Fetching apt key from SKS keyserver somehow failed. Trying to get one from alternative.\n'; wget $APTKEY_STORE_HTTPS -O - | sudo apt-key add -) || (echo 'Fetching apt key by an alternative method failed too. Exiting since ROS cannot be installed.'; error))
 lsb_release -a
-sudo apt-get update
+sudo apt-get update || (echo "ERROR: apt server not resonding. This is a rare situation but usually you have to wait for awhile (typically a few hours) then try again. Sorry for the inconvenience. See https://github.com/ros-industrial/industrial_ci/pull/56 for more of the discussion"; error)
```

Comment:
Suggestion: "a rare situation but usually" -> "a rare situation, and usually just waiting for a while clears this."